### PR TITLE
codelst:2.0.1

### PR DESCRIPTION
**Pull request by jneug**
Fixed codelst for Typst 0.11

I am submitting
- [ ] a new package
- [x] an update for a package

### CODE REVIEW

File: packages/preview/codelst/2.0.1/typst.toml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+name = "codelst"
+version = "2.0.1"
+entrypoint = "codelst.typ"
+authors = ["Jonas Neugebauer"]
+license = "MIT"
+description = "A typst package to render sourcecode."
+repository = "https://github.com/jneug/typst-codelst"
+exclude = ["example.typ", "example.pdf", "manual.pdf", "manual.typ", "tbump.toml"]
+compiler = "0.0.9"
+keywords = ["sourcecode", "code", "syntax-highlighting", "raw", "line numbers"]
```
Comment (laurmaedje) [inline packages/preview/codelst/2.0.1/typst.toml:11]: You might want to use the opportunity to add [`categories`](https://github.com/typst/packages/blob/main/CATEGORIES.md) and possibly also [`disciplines`](https://github.com/typst/packages/blob/main/DISCIPLINES.md).

File: packages/preview/codelst/2.0.1/typst.toml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+[package]
+name = "codelst"
+version = "2.0.1"
+entrypoint = "codelst.typ"
+authors = ["Jonas Neugebauer"]
+license = "MIT"
+description = "A typst package to render sourcecode."
+repository = "https://github.com/jneug/typst-codelst"
+exclude = ["example.typ", "example.pdf", "manual.pdf", "manual.typ", "tbump.toml"]
+compiler = "0.0.9"
```
Comment (laurmaedje) [inline packages/preview/codelst/2.0.1/typst.toml:10]: 0.0.9 is not a valid Typst compiler version. should be 0.9.0 or possible 0.11.0 if this has anything specific to 0.11+

Comment (drupol) [reply packages/preview/codelst/2.0.1/typst.toml:10]: Yes, this is specific to 0.11.0

Comment (jneug) [reply packages/preview/codelst/2.0.1/typst.toml]: Thanks for the heads-up. Should be fixed now.

File: packages/preview/codelst/2.0.1/typst.toml (hunk 1 above)
Comment (jneug) [reply packages/preview/codelst/2.0.1/typst.toml:13]: I just pushed a new typst.toml with the mentioned additions.

**Pull request merged**